foundations/build.rs

```diff
@@ -14,7 +14,7 @@ fn ensure_seccomp_sources_fetched() {
 
     if !src_dir.exists() {
         panic!(
-            "Can't find libssecomp sources. Run `git submodule update --init --recursive`. \
+            "Can't find libsseccomp sources. Run `git submodule update --init --recursive`. \
```

Comment:
While you're at it, you may as well fix the other typo? :)

Should be `libseccomp`

Reply:
🤦 

The typo fix has a typo. Of course it does.